engine/schema/mysql.sql

```diff
@@ -203,16 +185,16 @@ CREATE TABLE `prefix_queue` (
 
 -- activity stream
 CREATE TABLE `prefix_river` (
-  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `id` bigint(20) NOT NULL AUTO_INCREMENT,
   `type` varchar(8) NOT NULL,
   `subtype` varchar(32) NOT NULL,
   `action_type` varchar(32) NOT NULL,
   `access_id` int(11) NOT NULL,
```

Comment:
I have a better solution for this https://docs.google.com/a/elgg.org/document/d/1R3v_bYno6fw8mV5_GDW93uTI0pvR9S4yQ-BHMKTz1zs/edit#heading=h.xco4q1xouoxb